### ISSUE TITLE
Stuttering and lag throughout the game on all Mac's with Apple silicon processors despite high FPS

### ISSUE THREAD

**Issue by someone3753** [jeffhill/Dota2#1181]
All Apple silicon Mac's are very capable of playing DOTA with high framerates (even through the rosetta 2 translation layer) however there are stutters that happen throughout the game whenever there is any significant action such as spellcasting or item usage, etc.

It is an extremely frustraiting situation as I am getting consistent 120+ fps on my m1 pro macbook but the game is unplayable because everytime there is a critical teamfight or gank the game stutters and lags. I also had the same experience on my old m1 macbook air

I understand that this may be a limitation of using a translation layer to play DOTA but perhaps with the release of Apples new Metal 3 API and ARM processors becoming more prevalent something can be done to optimise DOTA for these machines?

**Comment by fonarelo** [comment]
Yeah. And it wasn't that bad before Valve did something with Spring Cleaning where they mentioned some M1 improvements.

**Comment by eazy12** [comment]
I think better to write it in Linux and Mac's issue tracker https://github.com/ValveSoftware/Dota-2/issues

Did you try running game with Proton? I am not sure is it available for Mac, but maybe it will work better because of DXVK and other additions.

**Comment by IgorHoruzha** [comment]
#15078 Possible the same issue. Do You still have the issue?

**Comment by fonarelo** [comment]
> #15078 Possible the same issue. Do You still have the issue?

There is none, if I'm using ethernet cable(not wi-fi) connection.
